enso/experiment/__init__.py

```diff
@@ -107,7 +107,7 @@ def experiment_has_been_run(self, current_settings):
         for col, val in current_settings.items():
             indexes = indexes & (results[col] == val)
         experiments = results.loc[indexes]
-        if len(experiments) < len(METRICS) * TEST_SETUP["n_splits"]:
+        if len(experiments) < (len(METRICS) + 2) * TEST_SETUP["n_splits"]:
```

Comment:
What's the `+2` for?

Reply:
Ah speed metrics. Got it.

Reply:
Yes, do you think speed metrics should be optional? There is very little overhead to running them so I think having them as default seems fine

Reply:
I'd say run as default